website/server/controllers/api-v3/groups.js

```diff
@@ -166,6 +166,12 @@ api.getGroup = {
     }
 
     group = Group.toJSONCleanChat(group, user);
+
+    if (group.purchased.plan.customerId) group.purchased.active = true;
```

Comment:
It is a pseudo field. The data is never saved. It is just used here to tell users without access to the customer information that the group does have an active subscription.